examples/qualcomm/utils.py

```diff
@@ -205,7 +205,7 @@ def execute(self, custom_runner_cmd=None, method_index=0):
             qnn_executor_runner_cmds = " ".join(
                 [
                     f"cd {self.workspace} &&",
-                    f"chmod +x ./qnn_executor_runner &&",
+                    "chmod +x ./qnn_executor_runner &&",
```

Comment:
is this change related?

Reply:
probably because of the lint runner I forgot to fix...

Reply:
Yeah I saw this on my prs and just bypassed it 😅

Reply:
Yeah lint runner failed on it.